tests/tensorflow/nightly/vision-coco.libsonnet

```diff
@@ -75,6 +86,15 @@ local utils = import 'templates/utils.libsonnet';
   },
   local v3_8 = tpu_common {
     accelerator: tpus.v3_8,
+    scriptConfig+: {
+      paramsOverride+: {
+        task+: {
+          train_data+: {
+            global_batch_size: 64,
```

Comment:
Does the default batch size not work for v3-8? If there's a constant train batch size per core/replica, it's better to compute the global batch size in the base object (`maskrcnn` in this case). Otherwise, we have to go through and set the global batch size manually for each case. See the validation batch size in `tpu_common` for an example.